auth/internal/delivery/grpc/login.go

```diff
@@ -0,0 +1,27 @@
+package auth_grpc
+
+import (
+	"context"
+	usecase_dto "msa_big_tech/auth/internal/usecases/dto"
+	auth "msa_big_tech/auth/pkg/proto/v1"
+)
+
+func (s *Implementation) Login(ctx context.Context, req *auth.LoginRequest) (*auth.LoginResponse, error) {
+
+	data := &usecase_dto.Login{
+		Email:    req.GetEmail(),
+		Password: req.Password,
```

Comment:
Лучше единообразно: либо используем гетеры либо поля.
```suggestion
		Password: req.GetPassword(),
```

Лучше использовать конечно гетеры, уберегают от неожтданных паник nil dereference:)